vortex-datafusion/src/persistent/opener.rs

```diff
@@ -58,23 +58,14 @@ impl VortexFileOpener {
 
         let projection = projection
             .as_ref()
-            .map(|fields| {
-                pack(
-                    fields.clone(),
-                    fields
-                        .iter()
-                        .map(|f| get_item(f.clone(), ident()))
-                        .collect(),
-                )
-            })
+            .map(|fields| Select::include_expr(fields.clone(), ident()))
```

Comment:
@joseph-isaacs this change seems to break TPCH, even though it seems like it should be the equivalent?

Reply:
fwiw you can do just `select` instead of `Select::`